content/_data/events/2019-12-03-Contributor-Summit.adoc

```diff
@@ -0,0 +1,5 @@
+title: "Jenkins Contributor Summit"
```

Comment:
The first line containing `---` is missing, so the header is not parsed properly.